templates/blog/topics/maas.html

```diff
@@ -0,0 +1,46 @@
+{% extends "templates/one-column.html" %} 
+
+{% block title %}MAAS{% endblock %}
+
+{% block meta_description %}All the latest news on MAAS (Metal as a Service) - the smartest way to manage bare metal.{% endblock %}
+
+{% block content %} 
+
+  <section class="p-strip--accent p-blog-topic__strip is-dark is-shallow u-image-position" style="overflow: hidden; background-color: #333">
```

Comment:
I spoke to @anasereijo to check this, we should use `is-deep` on topic pages rather than `is-shallow`, as it's closer to the height of the strip on the existing page.